src/ol/renderer/webgl/PointsLayer.js

```diff
@@ -320,7 +322,7 @@ class WebGLPointsLayerRenderer extends WebGLLayerRenderer {
               event.data.renderInstructions
             );
           }
-
+          this.ready = true;
```

Comment:
Shouldn't this only be set if the received data is *not* for hit detection?
Or after receiving both, if hitdetection is enabled?